docs/translations/zh/build/index.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This page is incorrect. I just downloaded the [file from CrowdIn](https://crowdin.com/project/ethereumfoundation/zh-CN#) (Contents v1.1). This is the content of the file:
```
---
title: 开始搭建
meta:
  - 
    name: 说明
    content: 通过Ethereum Studio了解以太坊基础知识。以太坊Studio是基于网页的IDE，可以用来搭建及测试智能合约，并为其搭建一个前端
  - 
    property: og:标题
    content: 开始搭建 | 以太坊
  - 
    property: og:说明
    content: 通过 Ethereum Studio 了解以太坊基础知识。Ethereum Studio 是基于网页的 IDE，可以用来搭建并测试智能合约。
sidebar: false
layout: 首页
hideHero: true
lang: zh-CN
---

<BuildPage />
```
Please ensure all files have been correctly downloaded.